backend/app/cmd/server.go

```diff
@@ -1341,5 +1343,5 @@ func (c *authRefreshCache) Get(key interface{}) (interface{}, bool) {
 
 // Set implements cache setter with key converted to string
 func (c *authRefreshCache) Set(key, value interface{}) {
-	_, _ = c.LoadingCache.Get(key.(string), func() (interface{}, error) { return value, nil })
+	_, _ = c.LoadingCache.Get(key.(string), func() (string, error) { return value.(string), nil })
```

Comment:
Do we need value interface{} here for real, or just a leftover ? the underlying cache is typed string now

Reply:
We do because go-pkg/auth, as of now, doesn't support generics and expects `interface{}`.